cypress/e2e/internal/notices/ad-hoc/returns-invitation.cy.js

```diff
@@ -0,0 +1,146 @@
+'use strict'
+
+import scenarioData from '../../../../support/scenarios/return-notices.js'
+
+describe('Ad-hoc returns invitation journey (internal)', () => {
+  beforeEach(() => {
+    cy.tearDown()
+    cy.calculatedDates().then((body) => {
+      const scenario = scenarioData(body.firstReturnPeriod)
+
+      cy.load(scenario)
+    })
+
+    cy.fixture('users.json').its('billingAndData').as('userEmail')
+    cy.fixture('notice-recipients.json').as('noticeRecipients')
+  })
+
+  it('invites a customer to submit returns', () => {
+    cy.get('@userEmail').then((userEmail) => {
+      cy.programmaticLogin({
+        email: userEmail
+      })
+    })
+
+    // Navigate to the Notices page
+    cy.visit('/system/notices')
+
+    // Start the standard notice journey
+    cy.get('.govuk-button').contains('Create an ad-hoc notice').click()
+
+    // Enter a licence number
+    cy.get('@noticeRecipients').then(({ licenceNumber }) => {
+      cy.get('#licenceRef').type(licenceNumber)
+    })
+    cy.get('button.govuk-button').click()
+
+    // Select the notice type
+    cy.get('#noticeType').check()
```

Comment:
You've got `#noticeType` and `#noticeType-2` further down. this means the radio options of this page don't have id's on the individual options which makes it harder to know what you're selecting without looking at the code. Other tests handled this by adding comments above each line which I felt was redundant. Code should document itself for basic things like this.
 
So if you add ids on the page this would become `#invitations` and `#reminders` for the one later. (The third one would be `#paperReturn` - basically just using the values as the id)